soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -725,8 +726,7 @@ void EnMag_DrawInnerMq(Actor* thisx, PlayState* play, Gfx** gfxp) {
 
         gDPPipeSync(gfx++);
         gDPSetPrimColor(gfx++, 0, 0, 255, 255, 255, (s16)this->subAlpha);
-        EnMag_DrawImageRGBA32(&gfx, 174, 145, gTitleMasterQuestSubtitleTex,
-                              EnMag_GetCopyrightTexWidth(), 32);
+        EnMag_DrawImageRGBA32(&gfx, 174, 145, gTitleMasterQuestSubtitleTex, 128, 32);
```

Comment:
why is this being changed to a hardcoded 128? it seems the logic before had it switching between 128 and 160 based on game version?

Reply:
`gTitleMasterQuestSubtitleTex` is not always 128 wide?
I don't know why they changed that in the first place.

Reply:
Looks like this might have actually been a mistake to begin with in the PAL 1.1 support PR, it's setting the width of the Master Quest Subtitle based on the Copyright text width.

Reply:
good deal, didn't realize that change had come in as part of the 1.1 pr

Reply:
Yeah that's my bad for missing it in review, probably for the same reason that Jack accidentally made that change to begin with lol.